net-proxy/3proxy/3proxy-0.8.8.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+EAPI=6
+inherit eutils
+
+DESCRIPTION="A really tiny cross-platform proxy servers set"
+HOMEPAGE="http://www.3proxy.ru/"
+SRC_URI="https://github.com/z3APA3A/3proxy/archive/${P}.tar.gz"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~x86 ~amd64 ~ppc"
+IUSE=""
+
+S="${WORKDIR}/${PN}-${P}"
+
+src_prepare() {
+	epatch "${FILESDIR}"/${P}-gentoo.patch
+	cp Makefile.Linux Makefile || die
+	default
+}
+
+src_install() {
+	local x
+
+	cd src || die
+	dobin 3proxy
+	for x in proxy socks ftppr pop3p tcppm udppm mycrypt dighosts icqpr smtpp; do
+		newbin ${x} ${PN}-${x}
+		[[ -f "${S}"/man/${x}.8 ]] && newman "${S}"/man/${x}.8 ${PN}-${x}.8
+	done
+	cd ..
```

Comment:
missing `|| die`
if you're cd'ing into a dir and from it again, consider using `pushd/popd >/dev/null`